packages/x-charts-pro/src/internals/plugins/useChartProZoom/ZoomConfig.selectors.ts

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from '@mui/x-charts/internals';
+import { selectorChartZoomState } from './useChartProZoom.selectors';
+import type { ZoomInteractionName, PanInteractionName } from './ZoomConfig.types';
+
+export const selectorZoomConfig = createSelector(
+  [selectorChartZoomState, (_state, interactionName: ZoomInteractionName) => interactionName],
+  (zoomState, interactionName) => zoomState.zoomConfig.zoom[interactionName] ?? null,
+);
```

Comment:
What about renaming `zoomConfig` by `zoomGestureConfig` or `zoomInteractionConfig`.

I fear that on the long run zoomConfig could look too close to zoomOptions and create confusion about which one to use for each usecase.